source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,395 @@
+#. Follow the steps in :ref:`bare-metal-install-server` to
+   install |CL| onto the VM virtual disk. Note:
 
+   #. In :guilabel:`Configure Installation Media`, navigate top
+      VBOX HARDDISK, and then select :guilabel:`Confirm`.
 
-Unmount the installation ISO
-============================
+   #. In :guilabel:`Advanced options` > :guilabel:`Manage User`, create an
+      administrative user.
 
-The |CL| installer ISO needs to be unmounted to allow the VM to boot from the
-virtual hard disk, which |CL| has been installed to.
+   #. Do not install the bundle `desktop-autostart`.
 
+#. When |CL| installation is complete, select :guilabel:`Exit`.
 
-#. Power off the |CL| VM.
+#. At the prompt, enter:
 
-   .. image:: ./figures/vbox/vbox-shutdown-vm.png
-      :alt: Powering off a VirtualBox VM
+   .. code-block:: bash
 
-#. Click *Settings* to configure the |CL| VM.
+      shutdown now
+
+Unmount the ISO
+===============
 
-   .. image:: ./figures/vbox/vbox-vm-created.png
-      :alt: A VM selected in VirtualBox Manager
+The |CL| installer ISO needs to be unmounted to allow the VM to boot from the
+virtual hard disk.
+
+#. Return to the |VBM|.
+
+#. Click *Settings* to configure the |CL| VM.
 
-#. From the *VM - Settings* window, navigate to the *Storage* pane from the 
+#. From the *VM - Settings* window, navigate to the *Storage* pane from the
    left-hand side.
 
-#. From the middle *Storage Devices* column, click the blue CD disk labeled  
-   *clear-<VERSION>-installer.iso* under the *Controller: IDE* from.
+#. From the middle *Storage Devices* column, click the blue CD disk labeled
+   *clear-<VERSION>-live-server.iso* under the *Controller: IDE* from.
+
+#. From the *Attributes* column at right, in *Optical Drive*, select the
+   blue CD icon beside and click *Remove Disk from Virtual Drive*.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Remove Disk from Virtual 
-   Drive*
+   .. figure:: figures/vbox/virtualbox-cl-installer-08.png
+      :scale: 100%
+      :alt: Remove Disk from Virtual Drive
 
-   .. image:: ./figures/vbox/vbox-vm-settings-unmount-ISO.png
-      :alt: Unmounting an ISO in VirtualBox VM Settings        
+      Figure 8: Remove Disk from Virtual Drive
 
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
+#. Click *OK* to exit the *VM Settings* menu and return to the main
    |VBM|.
 
+Install |VB| Linux Guest Additions
+==================================
+
+|CL| provides Linux Guest Additions drivers for full compatibility using an
+install script in the **kernel-lts** (Long Term Support) bundle by |CL|.
+
+#. From the |VBM| select the |CL| VM, and select :guilabel:`Start`.
+
+#. In the VM Console, log in as the administrative user previously created.
+
+#. A message may appear: "A kernel update is available: you may wish
```

Comment:
I meant to say to make this as a note just in case this message does appear and have the user reboot.  The way it is written doesn't really convey that message and a user might enter the reboot command even though they didn't need to.  Doesn't hurt anything just slows the process.  Hope that makes sense.  Everything else looks good!

Reply:
Done in latest commit. Please review and approve.